module_utils/network_lsr/argument_validator.py

```diff
@@ -1165,6 +1208,40 @@ def _validate_post(self, value, name, result):
                         % (result["type"]),
                     )
 
+            if result["type"] == "wireless":
```

Comment:
How about moving this in a separate method like `validate_wireless()?

Reply:
Have separated this out into a separate function now. Might be worth doing for the other connection types too.